src/main/resources/db/changelog/v.0.1/db.changelog-form.yaml

```diff
@@ -0,0 +1,62 @@
+databaseChangeLog:
+  - changeSet:
+      id: 31032021-add-form-table
+      author: alexei
+      changes:
+        - createTable:
+            tableName: form
+            columns:
+              - column:
+                  name: fm_id
+                  type: UUID
+                  constraints:
+                    nullable: false
+                    unique: true
+                    primaryKey: true
+              - column:
+                  name: fm_name
+                  type: VARCHAR(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: fm_lastname
+                  type: VARCHAR(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: fm_midlename
+                  type: VARCHAR(50)
+              - column:
+                  name: fm_email
+                  type: VARCHAR(50)
+              - column:
+                  name: fm_phone_number
+                  type: VARCHAR(50)
+              - column:
+                  name: fm_skype
+                  type: VARCHAR(50)
+              - column:
+                  name: fm_country
+                  type: VARCHAR(50)
+              - column:
+                  name: fm_city
+                  type: VARCHAR(50)
+              - column:
+                  name: fm_experience
+                  type: VARCHAR(255)
+              - column:
+                  name: fm_education
+                  type: VARCHAR(100)
+              - column:
+                  name: fm_file_path
+                  type: VARCHAR(100)
+              - column:
+                  name: fm_primary_skill
+                  type: VARCHAR(255)
+  - changeSet:
+      id: 31032021-insert-form-data
+      author: alexei
+      changes:
+        dbms: postgresql
+        sqlFile:
+          path: db/changelog/v.0.1/sqlFile/insert_form.sql
```

Comment:
As far as I understand, you want to fill the table with test data. 
To my mind it's better not to do this with liquibase, you can populate DB using Hibernate functionality. So, drop this changset, please.

Reply:
done